repograph/repograph.py

```diff
@@ -131,11 +143,18 @@ def main(token,
     repos = resolve_repos(gh, orgs, repos)
 
     for repo in repos:
-        for source, dest in iter_submodules(gh, repo):
+        for source, dest, comp in iter_submodules(gh, repo):
             if strip_org:
                 source = source.rsplit('/')[-1]
                 dest = dest.rsplit('/')[-1]
-            graph.add_edge(source, dest)
+            attrs = {}
+            if comp is not None:
+                attrs['label'] = str(comp.total_commits) \
```

Comment:
This might be more helpful if the string were something like `"{} commits behind master.".format(comp.total_commits)`